catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -625,7 +625,7 @@ public void setConvertToGroupItemSprite(boolean convertToGroupItemSprite) {
 	}
 
 	public boolean isBackgroundSprite() {
-		return look.getZIndex() == Constants.Z_INDEX_BACKGROUND;
+		return this == ProjectManager.getInstance().getCurrentlyEditedScene().getSpriteList().get(0);
```

Comment:
this is potentially going to cause problems. e.g. when in the stage the scene that is running now might not be the currently edited scene and consequently this could change the behaviour there. besides it is very likely that there are going to be situations where the `ProjectManager` might not have the currently edited scene set at all which again would lead to incorrect behaviour. 

I suggest we have another look at it.